prometheus_client/samples.py

```diff
@@ -34,6 +34,23 @@ def __lt__(self, other: "Timestamp") -> bool:
         return self.nsec < other.nsec if self.sec == other.sec else self.sec < other.sec
 
 
+class BucketSpan(NamedTuple):
```

Comment:
Can we put some docstrings on these types saying that they are experimental and for users to expect them to change at any time? I just don't want someone to import them expecting stability and for us to break them while we iterate.

Reply:
Sure, you had told me already, sorry. I added a warning on both those newly added classes.